providers/software_sources.rb

```diff
@@ -18,7 +18,15 @@
       
       current_lists = []
       remote_lists = []    
-      remote_lists.push("gecosv2.list")
+      
+      case $gecos_os
+      when "GECOS V3"
+        remote_lists.push("official-package-repositories.list")
```

Comment:
If I try to uninstall Mintsources, dependences take out "gecosws-artwork", "gecosws-cinnamon-conf", "python-software-properties" and "software-properties-gtk", but nothing about "mint-meta-core" or cinnamon. I think we can delete it and fix gecos dependences.